lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -276,41 +283,29 @@ def self.invoke_inline_ruby(aem, obj, inputs)
     def self.run_method(cmd)
       require 'open4'
       rc = nil
-      final_stderr = []
-      threads = []
       method_pid = nil
       begin
         status = Open4.popen4(*cmd) do |pid, stdin, stdout, stderr|
           method_pid = pid
           yield stdin if block_given?
           stdin.close
-          threads << Thread.new do
-            stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
-          end
-          threads << Thread.new do
-            stderr.each_line do |msg|
-              msg = msg.chomp
-              final_stderr << msg
-              $miq_ae_logger.error "Method STDERR: #{msg}"
-            end
-          end
-          threads.each(&:join)
+          stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
```

Comment:
@mkanoor [Here](https://github.com/ManageIQ/manageiq/pull/9958/files/890158aa81437769e355cb32f2fef666c0d48964#diff-869df78af525aa5e18c55ab3cb697e43R136) and [here](https://github.com/ManageIQ/manageiq/pull/9958/files/890158aa81437769e355cb32f2fef666c0d48964#diff-869df78af525aa5e18c55ab3cb697e43R137) I take Standard out and standard error within the process and redirect them to IoLoggers that will write to the log directly.  The only thing that should be on stdout or stderr at this point is anything that went wrong setting up the DRb session in the preamble.